app/src/main/res/layout/sheet_blood_sugar_entry.xml

```diff
@@ -104,9 +104,28 @@
         <TextView
           android:id="@+id/bloodSugarReadingUnitLabel"
           style="@style/Clinic.V2.MeasurementInputLayout"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginTop="@dimen/spacing_10"
+          android:labelFor="@+id/bloodSugarReadingEditText"
+          android:text="@string/bloodsugarentry_mg_dl"
+          android:textAppearance="@style/Clinic.V2.TextAppearance.Body2Left.Grey1"
+          android:visibility="gone"
+          app:layout_constraintEnd_toEndOf="@+id/bloodSugarReadingEditText"
+          app:layout_constraintStart_toStartOf="@+id/bloodSugarReadingEditText"
+          app:layout_constraintTop_toBottomOf="@+id/bloodSugarReadingEditText" />
+
+        <com.google.android.material.button.MaterialButton
+          android:id="@+id/bloodSugarReadingUnitButton"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
```

Comment:
The button height is a bit large compared to the mockups, can we create a new style for this button and adjust minimun height to 32dp.

In that style, we can override and provide a shape appearance overlay for setting the corner radius.

https://material.io/develop/android/theming/shape